src/Components/Footer/Footer.js

```diff
@@ -67,6 +68,14 @@ function Footer() {
             {' '}
             <FaTwitter />{' '}
           </a>
+          <a
+            data-testid="github"
+            href="https://github.com/Girl-Code-It/Opportunity-Calendar-Frontend"
```

Comment:
@Raj142 Please change the link to just 'https://github.com/Girl-Code-It'. Also, do take the latest pull before updating your PR.